packages/story-editor/src/components/library/panes/pageTemplates/pageTemplatesPane.js

```diff
@@ -78,6 +78,7 @@ function PageTemplatesPane(props) {
   const {
     actions: { getCustomPageTemplates },
   } = useAPI();
+  const canShowSaveTemplate = Boolean(getCustomPageTemplates);
```

Comment:
Maybe rename to `canSaveTemplate` or `supportsCustomTemplates`

Reply:
I couldn't come up with a better name, so I was expecting it in the code review 🙂 . `supportsCustomTemplates` looks good 👍